packages/viewer/example/example.html

```diff
@@ -1,74 +1,87 @@
 <!DOCTYPE html>
 <html>
+  <head>
```

Comment:
Honestly not sure why I touched this one 🤔 I think this only has formatting changes

Reply:
this looks like, it has some more changes, not sure where they are from